app/controllers/atmosphere/api/v1/appliance_types_controller.rb

```diff
@@ -93,6 +78,14 @@ def endpoint_payload
 
         private
 
+        def check_for_conflict!(appl)
+          vm = appl.virtual_machines.first
+          if vm && vm.state.saving?
+            raise Atmosphere::Conflict,
+                  'It is not allowed to save application twice'
```

Comment:
@mkasztelnik Hmm, I've often received PR comments regarding sections of code which were not actually part of the pending PR. Many of those were from you - and I've never made a fuss about it. Let's not overbureaucratize, shall we? ;)